source/SpinalHDL/Other language features/utils.rst

```diff
@@ -38,14 +38,14 @@ Many tools and utilities are present in :ref:`spinal.lib <lib_introduction>` but
 Cloning hardware datatypes
 --------------------------
 
-You can clone a given hardware data type by using the ``cloneOf(x)`` function. It will return you a new instance of the same Scala type and the same parameterization than ``x``.
+You can clone a given hardware data type by using the ``cloneOf(x)`` function. It will return a new instance of the same Scala type and the same parameterization as its argument.
```

Comment:
mm, I'm not happy with this phrase:
maybe cutting as its argument is best course of action here (and changing `parameterization  `with `parameters`)
so the result would be:
``` text
You can clone a given hardware data type by using the ``cloneOf(x)`` function. It will return a new instance of the same Scala type and same parameters.
```

Reply:
Yeah that works